test/spec/GeoExt/data/store/Layers.test.js

```diff
@@ -365,6 +365,56 @@ describe('GeoExt.data.store.Layers', function() {
 
         });
 
+        describe('synchronizePropertiesToMap', function() {
+            var map;
+            var layer;
+            beforeEach(function() {
+                layer = new ol.layer.Vector({
+                    source: new ol.source.Vector({
+                        features: [
+                            new ol.Feature()
+                        ]
+                    })
+                });
+                map = new ol.Map({
+                    layers: [layer]
+                });
+            });
+
+            it('by default it only synchronizes `title`', function() {
+                var store = Ext.create('GeoExt.data.store.Layers', {map: map});
+
+                var layerRec = store.getAt(0);
+                layerRec.set('title', 'Kalle Berga');
```

Comment:
:+1:

Reply:
Never thought this would make it that long in the code :see_no_evil:  :rofl:

Reply:
... i just copied it from the example out of your issue :upside_down_face: thought it would be as good as any other value.

Reply:
That's a perfect value :+1: :laughing: